config/bootstrap.php

```diff
@@ -33,11 +33,21 @@
  */
 require CORE_PATH . 'config' . DS . 'bootstrap.php';
 
+// You can remove this if you are confident that your PHP version is sufficient.
+if (version_compare(PHP_VERSION, '5.5.9') < 0) {
```

Comment:
People concerned about performance can delete these lines for even bigger speed boosts :smile: